app/helpers/lessons_helper.rb

```diff
@@ -0,0 +1,15 @@
+module LessonsHelper
+  def course_url(lesson)
```

Comment:
You're right. The view was a bit cluttered, the only motivation to move this to the helper was to make the view code readable :smile: